api/models/node.go

```diff
@@ -35,6 +35,53 @@ func (i IndexList) Trace(span opentracing.Span) {
 func (i IndexList) TraceDebug(span opentracing.Span) {
 }
 
+type IndexTagFindSeries struct {
+	OrgId int      `json:"orgId" binding:"Required"`
+	Expr  []string `json:"expressions"`
+	From  int64    `json:"from"`
+}
+
+func (t IndexTagFindSeries) Trace(span opentracing.Span) {
+	span.SetTag("org", t.OrgId)
+	span.SetTag("expressions", t.Expr)
+	span.SetTag("from", t.From)
+}
+
+func (i IndexTagFindSeries) TraceDebug(span opentracing.Span) {
+}
+
+type IndexTagDetails struct {
+	OrgId  int    `json:"orgId" binding:"Required"`
+	Filter string `json:"filter"`
+	Tag    string `json:"tag"`
```

Comment:
are empty tag keys legal? otherwise add `binding:"Required"` here

Reply:
you're right, that should be required https://github.com/grafana/metrictank/pull/750/commits/5deda8aea2603aa694abc9009aeb26b881bcddaf